inbox/interruptible_threading.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         thread is started. Otherwise, the subclass must implement the `_run`
         method.
         """
+        self._should_be_interrupted = threading.Event()
```

Comment:
nit: shouldn't we use double underscores for naming consistency?

Reply:
It's because it's not really private to the class, the free functions are using it.